modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/winback/WinbackViewModel.kt

```diff
@@ -509,7 +511,19 @@ internal data class SubscriptionPlan(
     val title: String,
     val formattedPrice: String,
     val billingPeriod: BillingPeriod,
-)
+    val basePrice: BigDecimal,
+    val currencyCode: String,
+) {
+    val pricePerWeek = basePrice
+        .let { price ->
+            val weeksCount = when (billingPeriod) {
+                BillingPeriod.Monthly -> 4
```

Comment:
I thought this was an issue but we aren't displaying the monthly week price. But just in case we do later. Should the monthly price be calculated by multiplying the price by 12 and then dividing by 52?

$6.49 per month / 4 = $1.6225
or
$6.49 per month  × 12 / 52 = $1.4977

Reply:
Yeah, makes sense. I'll update the code. Thanks!